src/libstd/io/mod.rs

```diff
@@ -1425,6 +1425,12 @@ pub trait BufRead: Read {
     ///     println!("{}", line.unwrap());
     /// }
     /// ```
+    ///
+    /// # Errors
+    ///
+    /// Each line of the iterator has the same error semantics as [`BufRead::read_line()`].
```

Comment:
Not a big deal, but in the future, you don't need to add `()` in the method name. Not too concerned with this right now since we'll probably remove them all in bulk sometime in the future.